doc/dependencies.R

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this expected?

Reply:
Yes, the vignette has no R code, which is why there is nothing here